repo_state.yaml

```diff
@@ -0,0 +1,32 @@
+makefile:
```

Comment:
Is the plan to make this a part of the repo? I though we were gonna consolidate these somewhere centrally?

Reply:
I added this here to show what the report looks like currently.  I'll remove this after I have finished with initial input